tools/pipelines/templates/build-npm-package.yml

```diff
@@ -146,7 +146,7 @@ variables:
       releaseBuildOverride: ${{ parameters.releaseBuildOverride }}
       buildNumberInPatch: ${{ parameters.buildNumberInPatch }}
       nonScopedPackages: ${{ parameters.nonScopedPackages }}
-  - name: toolAbsolutePath
+  - name: absolutePathToTelemetryGenerator
```

Comment:
Renamed for extra clarity

Reply:
Could we add docs to it while we're here? 😁